src/nvim/channel.c

```diff
@@ -0,0 +1,512 @@
+// This is an open source non-commercial project. Dear PVS-Studio, please check
```

Comment:
No. channels are a higher-level abstraction, for vimL (and future lua) consumption, above the low-level libuv-wrapping `event/` code.